invenio_rdm_records/services.py

```diff
@@ -8,15 +8,33 @@
 
 """Bibliographic Record Service."""
 
-from invenio_records_resources.services import RecordService, \
-    RecordServiceConfig
+from invenio_drafts_resources.services import RecordDraftService, \
+    RecordDraftServiceConfig
+from invenio_records_resources.services import MarshmallowDataValidator
 
-# TODO: IMPLEMENT ME if more specialized configuration needed
-# class BibliographicRecordServiceConfig(RecordServiceConfig):
-#     """Bibliographic Record Service configuration."""
+from .marshmallow import MetadataSchemaV1
+from .models import BibliographicRecord, BibliographicRecordDraft
+from .permissions import RDMRecordPermissionPolicy
 
 
-class BibliographicRecordService(RecordService):
-    """Bibliographic Record Service."""
+class BibliogrpahicRecordDraftServiceConfig(RecordDraftServiceConfig):
```

Comment:
Same discussion as above and if naming is done later, then there is a small typo anyway:

```suggestion
class BibliographicRecordDraftServiceConfig(RecordDraftServiceConfig):
```